### ISSUE TITLE
Проверка текстов запросов в модулях без установки Language Tool

### ISSUE THREAD

**Issue by DoublesunRUS** [1C-Company/1c-edt-issues#240]
### Описание проблемы

Плагин v8-code-style умеет выполнять проверки текстов запросов.
К сожалению чтобы он проверял тексты запросов указанных в модулях, необходимо установить плагин Language Tool.

### Описание решения проблемы

Перенести функционал из плагина Language Tool в EDT в той части, которая добавляет функционал проверки текстов запросов в модулях.

**Comment by marmyshev** [comment]
Техническое пояснение к задаче.

Перенос проверки-хаба из LT в ядро ЕДТ - требует решения 2х задач:
1. поиск всех строковых литералов с текстами запросов - либо перенести из LT библиотеку типизации/категоризации строковых литералов, либо написать свой "велосипед". Это не простая задача, как кажется с первого взгляда, потому что есть очень много разных типов (категорий) строковых литералов (`StringLiteral`). Сейчас в 1C:Language Tool есть система типизации/категоризации строковых литералов - была сделана для целей перевода кода в первую очередь, которая довольно точно определяет тип "`Запрос`". На основе этой типизации литералов сделана в LT проверка-хаб позволяющая запускать проверки модели языка запросов.
2. оптимальная загрузка по тексту запроса вторичной модели языка запросов - либо перенести из LT общий сервис загрузки модели, либо написать свой "велосипед"

**Comment by tretyakevich** [comment]
Пока предварительно ставим на 2022.1, но зависит от результатов анализа указанной в пункте 1 задачи.

**Comment by Jimmo910** [comment]
оч ждем

**Comment by MaksimDzyuba** [comment]
@Jimmo910 Здравствуйте, так она же доступна, вот отдельный хаб или что имелось ввиду

<img width="1018" height="820" alt="Image" src="https://github.com/user-attachments/assets/e99c5e26-8177-4362-869e-b2285a9586bd" />

**Comment by Jimmo910** [comment]
@MaksimDzyuba ишью то открыт... мы ж не знаем об этом...

**Comment by MaksimDzyuba** [comment]
@Jimmo910 как попробуете, отпишитесь, пожалуйста, работает ли так, как ожидали

**Comment by Jimmo910** [comment]
@MaksimDzyuba не знаю так ли подрузомивалось. но полей в выборке не появилось. зато начал ругаться на то что в расширении не добавлено.  хотя как добавить ВАЛАДЕЛЬЦА ????

<img width="1558" height="957" alt="Image" src="https://github.com/user-attachments/assets/d533f39f-3f15-4f50-8cf3-51c9eed5b5eb" />

**Comment by MaksimDzyuba** [comment]
@Jimmo910 да, поля и не проявляются, этого не было в переносе, только валидация, ну как в заголовке и написано. Поля остались именно в lang-tool, как часть для переводов, плюс там с производительностью сложно

**Comment by Jimmo910** [comment]
@MaksimDzyuba т.е. если хотим еще и видить поля запроса в выборке - то лучше ставить плагин - верно?

**Comment by MaksimDzyuba** [comment]
ну да, я просто не уверен, что мы задачу эту сделаем и производительность не просядет в ближайшее время

**Comment by Jimmo910** [comment]
@MaksimDzyuba так получается могу подождать и в будущих версия все таки будет сразу в ЕДТ возможночть видеть поля выборки запроса? а то я запутался)
